src/kernel/include/shared/cos_types.h

```diff
@@ -370,6 +378,16 @@ typedef unsigned long      invtoken_t;
 #define THDCLOSURE_INIT
 typedef int                thdclosure_index_t;
 
+/* This is an attempt decouple hardware specific code from
+   parts of the kernel interface that are kernel agnostic.
```

Comment:
Multiline comments:

```
/*
 * comment here...
 * continuing here...
 */
```

Reply:
This enables the use of `/**` and `/***` to emphasize function/high-level comments, and module-level comments.